src/app/App.tsx

```diff
@@ -1,9 +1,10 @@
 import React from 'react'
+import LocationInput from './components/LocationInput/LocationInput'
 
-import { BrowserRouter } from 'react-router-dom'
+//import { BrowserRouter } from 'react-router-dom'
```

Comment:
Maybe it would be better to delete this instead of commenting it out before deploying.

Reply:
also very true, ill dlete it with th next commit :)